app/indexers/indexes_metadata.rb

```diff
@@ -6,17 +6,16 @@ def generate_solr_document
       solr_doc['title_ssi'] = object.title.first
       solr_doc['date_created_ssi'] = object.date_created.first
       solr_doc['creator_tesim'] = creator_alternate_names(object).to_a + creator_names(object).to_a
-      solr_doc['alpha_creator_tesim'] = creator_names(object).sort
+      solr_doc['alpha_creator_tesim'] = creator_names(object).reject(&:blank?).sort
```

Comment:
Could we move the `.reject...` logic into the `creator_names` method?  I think it would make this line a little easier to read without adding to much overhead to the method.

Reply:
done